src/lxc/cgroups/cgfsng.c

```diff
@@ -987,6 +996,10 @@ static void lxc_cgfsng_print_handler_data(const struct cgfsng_handler_data *d)
 	printf("  container name: %s\n", d->name ? d->name : "(null)");
 	printf("  lxc.cgroup.use: %s\n", cgroup_use ? cgroup_use : "(null)");
 	printf("  lxc.cgroup.pattern: %s\n", d->cgroup_pattern ? d->cgroup_pattern : "(null)");
+	if (d->cgroup_meta.dir)
+		printf("  lxc.cgroup.dir: %s\n", d->cgroup_meta.dir);
+	else
```

Comment:
...  are you doing this (instead of the same thing the previous 3 lines do) to stay under 80 cols?

Reply:
Yeah

Reply:
replaced with `cond ? x : y`